src/components/InventoryDetailsPopover.vue

```diff
@@ -4,19 +4,19 @@
       <ion-list-header>{{ translate("Inventory computation")}}</ion-list-header>
       <ion-item>
         <ion-label class="ion-text-wrap">{{ translate("Quantity on hands")}}</ion-label>
-        <ion-note slot="end">52</ion-note>
+        <ion-note slot="end">{{ getProductStock(item.productId)?.quantityOnHandTotal ?? '0' }}</ion-note>
```

Comment:
```suggestion
        <ion-note slot="end">{{ getProductStock(item.productId).quantityOnHandTotal ?? '0' }}</ion-note>
```

No need to add optional chaining as we are returning an object and accessing any value on empty object will return undefined but will not throw any error.